acct-user/miniflux/miniflux-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-user
+
+DESCRIPTION="User for www-apps/miniflux"
+ACCT_USER_ID=-1
```

Comment:
I'd add a comment here indicating you know it won't actually be -1 (forbidden in ::gentoo).

They need to be reserved via the uid-gid.txt file (where is it?) first. I figure you know this (although it's fine if you don't) but would rather say than not.

Reply:
Whoops. I forgot to set this back to 404 when moving this from my test environment.

Reply:
Please see: https://github.com/gentoo/api-gentoo-org/pull/509 :)

Reply:
excellent